app/lib/features/chat_ng/providers/chat_room_messages_provider.dart

```diff
@@ -49,3 +53,39 @@ final renderableChatMessagesProvider =
     return _supportedTypes.contains(msg.eventItem()?.eventType());
   }).toList();
 });
+
+final mentionSuggestionsProvider =
+    StateProvider.family<Map<String, String>?, (String, MentionType)>(
+        (ref, params) {
+  final roomId = params.$1;
+  final mentionType = params.$2;
+  final client = ref.watch(alwaysClientProvider);
+  final userId = client.userId().toString();
```

Comment:
we have a provider for that:
```suggestion
  final userId = ref.watch(ref.watch(myUserIdStrProvider));
```

Reply:
Oops, yes. Hard to track too many providers... ^^!